eduNEXT/xblock-filesmanager
# fix adding visibility file dates in the actions menu

### Description 

This PR adds the actions for setting visibility dates to the 'Actions' menu. When you right-click on a file/folder, you can see the buttons 'Set file visibility with date' and 'Remove date visibility', respectively. Now, they are available in the 'Actions' menu as well


BEFORE         | NOW
:-------------------------:|:-------------------------:
![](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/54f0331c-f382-4660-b850-1efd176d7b41)  |  ![](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/4639ecf3-47d1-4a1a-8070-2d698d491c8c)



### How to test it

- Check this branch`jv/fix-adding-buttons-to-actions-menu` in the XBlock files manager

1. use node 18
 ```
nvm use
```

2. install dependencies
```
yarn install 
```
3 run to see any change in the react code 
```
yarn dev 

```

#### Setting/Removing dates visibility

1. Click on a file/folder then click on 'Actions' button on the Menu
![click-test](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/9d41f5c2-4f6f-4c52-bacc-2022ff9a17a7)

2. You should see the options  'Set file visibility with date' and 'Remove date visibility'
![options-test](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/a7ddcf5d-eaed-41dc-9ab2-9acc8eb795ea)




**Merge checklist:**
Check off if complete *or* not applicable:
